### DISCUSSION TITLE
Multiple go-e and priorization

### DISCUSSION THREAD

**Discussion by MiOrt** [marq24/ha-goecharger-api2#27] [Q&A]
Hi, at the moment I do have an openWB controlling my go-e (two loadpoints, one each). I now found the go-e integration and was thinking about replacing the openWB by a second go-e. Now how does the integration handle this? Will this result in a 50/50 power per go-e, or can I prioritise one go-e over the other (especially in PV mode)?

As far as I know... if I configure more than one charger in the app, there is a priority-function available.

**Comment by marq24** [comment]
The Integration does not have any "internal" functionality... So what ever is possible to configure/setup via the go-eCharger API v2 will be possible to configure via HA (except: https://github.com/marq24/ha-goecharger-api2?tab=readme-ov-file#list-of-currently-not-handled-api-keys-24172)... 

No clue what api keys actually control the priority of two chargers... I guess this are this ones (which can be configured via the integration)

lop | R/W | uint16 | Config | load_priority
-- | -- | -- | -- | --
lof | R/W | uint8 | Config | load_fallback

**Comment by MiOrt** [comment]
I asked go-e for help on those two API-keys and received a helpful answer:

Both API keys are indeed purely related to load management in this case, as you have already correctly observed.

Brief explanation of load management: With load management, you can group multiple go-eChargers into a load management group and set a total current limit. The priority of the chargers can be freely configured, allowing the current to be distributed either fairly or sequentially between the chargers.

To use static or dynamic load management (dynamic load management requires a go-e Controller), an active internet connection and cloud access are necessary.

Therefore, it is indeed the case that these two API keys will have no impact on PV surplus charging but are solely used for load management.
